source/renderer/app/components/sidebar/Sidebar.js

```diff
@@ -28,12 +31,23 @@ type Props = {
   isIncentivizedTestnet: boolean,
 };
 
-export type SidebarMenus = ?{
-  wallets: {
-    items: Array<SidebarWalletType>,
-    activeWalletId: ?string,
-    actions: {
-      onWalletItemClick: Function,
+export type SidebarMenus = {
+  nonHardwareWallets: ?{
+    wallets: {
+      items: Array<SidebarWalletType>,
+      activeWalletId: ?string,
+      actions: {
+        onWalletItemClick: Function,
+      },
+    },
+  },
+  hardwareWallets: ?{
+    wallets: {
+      items: Array<SidebarHardwareWalletType>,
+      activeWalletId: ?string,
+      actions: {
+        onHardwareWalletItemClick: Function,
+      },
```

Comment:
@DeeJayElly can we just have SidebarMenus with `wallets` and `hardwareWallets` without unnecessary nested structure?

Reply:
Fixed